### DISCUSSION TITLE
Disable Updating of Devices

### DISCUSSION THREAD

**Discussion by t8simon** [bb-Ricardo/netbox-sync#213] [Q&A] [answered]
Hi

I track all devices already in netbox with a certain system and netbox-sync would change a lot on that (also because of legacy in the our vmware-setup). 

So basically I want the hardware devices to stay untouched but all the VMs (with their IPs and interfaces etc) synced into netbox. 

What would be the best way to achieve this? 

Thanks in advance
Simon

**Comment by bb-Ricardo** [comment, marked as answer]
Hi, you could just use the host exclude filter and exclude all host. Something like this:

```ini
# This will only include/exclude the host, not the VM if Host is part of a multi host
# cluster.
#host_exclude_filter =
host_include_filter = .*
```

Just try this one.

**Reply by t8simon** [reply]
Nice, that did the trick! Easier than I thought :) Must have overread that one in the settings file. 
Thanks!
